docs/CodeOwners.md

```diff
@@ -0,0 +1,112 @@
+# Code Owners
+
+This file outlines the owners of the FHIR Server features in OSS.
+
+### Portal
+
+rojo (RB Johnson)
+
+patalada (Paul Taladay)
+
+### Custom Search & reindexing
+
+jaerwin (Jared Erwin)
+
+rotodd (Robin Todd)
+
+### Patient $everything
+
+rotodd (Robin Todd)
+
+ivmatan (Ivan Matantsev)
+
+### Sort
+
+namadabu (Narasimhan Madabusi)
+
+### Schema management
+
+ribans (Richa Bansal)
+
+apurvabhale (Apurva Bhale)
+
+### Disable History
+
+rotodd (Robin Todd)
+
+patalada (Paul Taladay)
+
+### SQL perfomance
+
+ivmatan (Ivan Matantsev)
+
+ribans (Richa Bansal)
+
+jaerwin (Jared Erwin)
+
+### SQL CRUD operations
+
+rotodd (Robin Todd)
+
+### SQL search
+
+namadabu (Narasimhan Madabusi)
+
+### Cosmos DB performance
+
+bkowitz (Brendan Kowitz)
+
+jaerwin (Jared Erwin)
+
+ivmatan (Ivan Matantsev)
+
+### Cosmos CRUD operations
+
+bkowitz (Brendan Kowitz)
+
+jaerwin (Jared Erwin)
+
+### Cosmos search
+
+bkowitz (Brendan Kowitz)
+
+jaerwin (Jared Erwin)
+
+### Profiles & validation
+
+ivmatan (Ivan Matantsev)
+
+### Bulk export
+
+namadabu (Narasimhan Madabusi)
+
+rojo (RB Johnson)
+
+### Chained search
+
+bkowitz (Brendan Kowitz)
+
+### Conditional delete
+
+bkowitz (Brendan Kowitz)
+
+### Purge history
+
+bkowitz (Brendan Kowitz)
+
+### Patch
+
+ivmatan (Ivan Matantsev)
+
+bkowitz (Brendan Kowitz)
+
+### Bulk import
+
+tongwu (Tong Wu)
+
+peizhou (Peiyuan Zhou)
+
+### Resource change capture
+
+teli (Teng Li)
+
```

Comment:
Should we have sections for the non-FHIR server parts of our Github repo?
Like for the build, release, samples, and tools folders?